packages/documentation/src/stories/components/popover/popover.stories.ts

```diff
@@ -30,6 +30,7 @@ const meta: MetaComponent = {
     placement: 'top',
     arrow: true,
     title: true,
+    maxWidth: '80vw',
```

Comment:
Should we leave this empty by default so the popover uses the default 280px max-width if nobody tweaks it?